### PR TITLE
order argument pin & secret and typo

### PR DESCRIPTION
**Pull request by minscof**

### CODE REVIEW

**Comment by minscof** [comment]
add typo instalation

File: libetrv/device.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def __new__(mcls, name, bases, attrs):
 
 
 class eTRVDevice(metaclass=eTRVDeviceMeta):
-    def __init__(self, address, secret=None, pin=None):
+    def __init__(self, address, pin=None, secret=None):
```
Comment (AdamStrojek) [inline libetrv/device.py]: What is the purpose of changing the order of these parameters?

**Comment by boelle** [comment]
@minscof AdamStrojek is not sure what the purpose of the change is, could you maybe explain?

**Comment by AdamStrojek** [comment]
No answer on the question that was proposing this change

**Pull request closed without merging**